kss-builder/index.hbs

```diff
@@ -212,14 +212,12 @@
 <footer role="contentinfo">
   <div class="wrapper">
     <div class="footer-logo">
-      <img src="https://www.gov.au/alpha/assets/coatofarms-01.svg" alt="Australian Government Coat of Arms" />
+      <img alt="Australian Government Coat of Arms" src="../assets/img/logos/coatofarms-01.svg">
```

Comment:
Let's keep the image src relative for now - once this feature is deployed we can adjust image src to the absolute url.